### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by ghost** [ruffle-rs/ruffle#4863]
Not working on Linux. Hardware acceleration is required.

**Comment by relrelb** [comment]
Can you please describe what's exactly not working? Did you encounter a specific SWF that doesn't work well? Without this information we're unable to address the problem.

**Comment by torokati44** [comment]
Yes, at the moment, a Vulkan implementation is required to run Ruffle on Linux.
You can try running it with Lavapipe though, which is a pure software Vulkan implementation.

**Comment by torokati44** [comment]
Or you can use the browser version. Either the extension (which also runs direct links to .swf files), or this page: https://ruffle.rs/demo/
These can work with or without WebGL just as well.
The performance is not that good as with a native build though.

**Comment by artenaki** [comment]
Thanks. I'll try alternative ways.

**Comment by Lord-McSweeney** [comment]
Closing in favor of #672

**Comment by kjarosh** [comment]
Based on my experience Ruffle does work very well without hardware acceleration on Linux (lavapipe/llvmpipe). Please let us know if that's not the case.
